data/interfaces.txt

```diff
@@ -18,6 +18,8 @@ color "bright" .75 .75 .75 0.
 color "medium" .5 .5 .5 0.
 color "dim" .25 .25 .25 0.
 color "faint" .1 .1 .1 0.
+color "special" .5 .5 .1 0.
+color "dead" .4 0. 0. 0.
```

Comment:
These should be moved to their own "block" and probably also given more scoped names.